src/containers/ProjectContentsContainer.jsx

```diff
@@ -53,6 +59,9 @@ class ProjectContentsContainer extends Component {
           <ModalBody>
             <p>{this.state.translationText}</p>
             <input autoFocus placeholder="Translate some text" />
+            <DefaultButton onClick={this.handleSubmit}>
+              Submit
+            </DefaultButton>
           </ModalBody>
           <ModalFooter>
             <DefaultButton onClick={() => this.setState({modalOpen: false})}>
```

Comment:
OK, I like this last idea better. Atm I'm having trouble even running the project, after I've switch to node 8 :/